src/plugins/mer/merplugin.cpp

```diff
@@ -154,6 +154,16 @@ bool MerPlugin::initialize(const QStringList &arguments, QString *errorString)
     RunControl::registerWorker<RemoteLinuxQmlProfilerSupport>(QML_PROFILER_RUN_MODE, constraint);
     //RunControl::registerWorker<RemoteLinuxPerfSupport>(PERFPROFILER_RUN_MODE, constraint);
 
+    Utils::globalMacroExpander()->registerVariable("SailfishOSVariant",
+                                                   tr("Category of a wizard for %1 files").arg(Sdk::osVariant()),
```

Comment:
Actually the possible use is not limited for wizard category. So e.g. `"Expands to \"Sailfish OS\" or its variant name"`, `"Expands to \"Sailfish IDE\" or its variant name"`... would be more appropriate.

The variables should be SailfishOsVariant, SailfishIdeVariant and SailfishSdkVariant (notice the case) unless I missed something.